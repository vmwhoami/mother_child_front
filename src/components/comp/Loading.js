import React from 'react';
import { Heart } from 'react-awesome-spinners';

const Loading = () => {
  const heart = { color: '#10BBB5' };
  return (
    <div className="container">
      <span>Loading ...</span>
      <Heart color={heart.color} />
      {' '}
      :
    </div>
  );
};

export default Loading;
