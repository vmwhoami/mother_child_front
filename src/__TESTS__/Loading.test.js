import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../components/comp/Loading';

it('renders Loading  correctly', () => {
  const tree = renderer.create(<Loading />);
  expect(tree).toMatchSnapshot();
});