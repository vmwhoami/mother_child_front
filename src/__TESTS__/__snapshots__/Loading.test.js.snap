// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loading  correctly 1`] = `
<div
  className="container"
>
  <div
    className="loading"
  >
    <span>
      Loading ...
    </span>
    <div
      className="sc-jSgupP Cjkqn"
      color="#10BBB5"
      size={46}
    >
      <div />
    </div>
  </div>
</div>
`;
